actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -206,6 +206,11 @@ def excerpt(text, phrase, options = {})
       # +plural+ is supplied, it will use that when count is > 1, otherwise
       # it will use the Inflector to determine the plural form.
       #
+      # If passed an optional +locale+ parameter, the word will be pluralized
```

Comment:
Makes sense! (Wouldn't be a trailing colon – `locale:` – even more exact? Don't really know the conventions for documenting named arguments, though …)